### DISCUSSION TITLE
Does the ending year of copyright need to be updated?

### DISCUSSION THREAD

**Discussion by leleliu008** [universal-ctags/ctags#3692] [General]
![Screenshot from 2023-04-06 14-50-52](https://user-images.githubusercontent.com/3040852/230294324-219a9841-99d2-4085-ad65-b5686525c9c5.png)

@masatake I suddenly discovered that the ending year of copyright in our source code is still 2022, does this need to be updated?

**Comment by masatake** [comment]
Yes, I think we have to update them.
